coding_systems/versioning/tests/test_models.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime, timezone
+
+import pytest
+from django.db import connections
+from django.db.utils import OperationalError
+from django.utils.connection import ConnectionDoesNotExist
+
+from coding_systems.snomedct.models import Concept
+from coding_systems.versioning.models import (
+    CodingSystemVersion,
+    update_coding_system_database_connections,
+)
+
+
+def test_coding_system_version_db_name(coding_system_version):
+    assert coding_system_version.db_name == "snomedct_245584db"
+
+
+def test_coding_system_version_latest(coding_system_version):
+    # latest depends on valid_from date, not imported date
+    # make a later import, but for an earlier version
+    new_cs_version = CodingSystemVersion.objects.create(
+        coding_system="snomedct",
+        version="v0.1",
+        import_ref="ref",
+        valid_from=datetime(2022, 9, 1, tzinfo=timezone.utc),
+    )
+    # latest is coding system-dependent, so a later valid_from version
+    # for a different coding system is irrelevant
+    CodingSystemVersion.objects.create(
+        coding_system="dmd",
+        version="v1",
+        import_ref="ref",
+        valid_from=datetime(2022, 10, 15, tzinfo=timezone.utc),
+    )
+
+    assert new_cs_version.import_timestamp > coding_system_version.import_timestamp
+    assert CodingSystemVersion.latest("snomedct") == coding_system_version
+
+    # an unknown coding system
+    assert CodingSystemVersion.latest("foo") is None
```

Comment:
I wasn't sure - a `Codelist` has a `coding_system` property too, which I think should return the most recent version.  But it can legitimately be None, if the codelist is opcs4 one, so it either needs to check if it's None, or catch an expected exception.  Is the latter bettter?